### DISCUSSION TITLE
CommunityToolkit DataGrid DataGridComboBoxColumn Value - Display Member Issue

### DISCUSSION THREAD

**Discussion by esobrino** [unoplatform/uno#8911] [Q&A]
Given a ContentList that has:

- Description 
- IdentifierId
- ...  (other properties not needed here)

The DataGrid ItemsSource bounded class we have:

- ContentIdentifierId
- ... (other properties not needed here)

```
            <tk:DataGridComboBoxColumn ItemsSource="{x:Bind ContentList, Mode=OneWay}" 
                                       Header="Topic" DisplayMemberPath="Description"
                                       Binding="{Binding ContentIdentifierId}" />
```
It looks like the only way to make this work is that the ContentList items and the Grid's ItemsSource have a property with the same name.  What are the alternatives when they don't.  I had work with other data-grids in WPF and there is always a way to setup what is sometimes called "ValueMemberPath", I spend time looking for answers without success.

Can someone provide some alternatives to solve this issue?   help is greatly appreciated.

**Comment by jeromelaban** [comment]
Thanks for the report. Is this behavior you're observing also present when running on UWP or WinAppSDK (the `.UWP` or `.Windows` project in the uno project template) on Windows?

**Reply by esobrino** [reply]
Sorry, forgot to provide that detail...  I am using the CommunityToolkit.WinUI.UI.Controls...

**Reply by jeromelaban** [reply]
Thanks. This may be a good question to ask here then: https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions, as it's the home for the DataGrid control.

**Reply by esobrino** [reply]
Just did, thanks!
